development/src/logic/pieces.ts

```diff
@@ -89,7 +93,10 @@ export class Pawn extends Piece {
   enPassant: boolean;
 
   constructor(position: Position, player: Player) {
-    super(position, player, pawnResource, 'Pawn');
+    const logo = player == whitePlayer
```

Comment:
I think it would be better to remove the `export` on `whitePlayer` & `blackPlayer` and change the if to be `if (player.color === PlayerColor.White)`

Reply:
Fixed